docs/getting-started.md

```diff
@@ -26,8 +26,22 @@ Check out the [Local Environment](./local-environment.md) document.
 
 First of all, you need to make sure that all PHP and JavaScript dependencies are installed:
 
+Install Composer by following [installation instructions](https://getcomposer.org/download/), and then ensure to downgrade to version 1:
+
+```bash
+composer require composer/composer:~1.10.16
```

Comment:
```suggestion
```

This is not needed. `composer self-update --1` is all that's needed once installed via above instructions

Reply:
Got it, done!